README.md

```diff
@@ -1,20 +1,17 @@
-[![version(scoped)](https://img.shields.io/npm/v/openapi-typescript-fetch.svg)](https://www.npmjs.com/package/openapi-typescript-fetch)
-[![codecov](https://codecov.io/gh/ajaishankar/openapi-typescript-fetch/branch/main/graph/badge.svg?token=Z8GQ6M5KAR)](https://codecov.io/gh/ajaishankar/openapi-typescript-fetch)
-
-# 📘️ openapi-typescript-fetch-sveltekit
+# 📘️ apity - Typed API client for Svelte and SvelteKit
 
 A typed fetch client for [openapi-typescript](https://github.com/drwpow/openapi-typescript) compatible with SvelteKit's custom `fetch`
```

Comment:
The top section should list the design goals and overall capabilities, e.g. the `.reload`, having the caching (in future), etc.